src/OsuSharp/Entities/Replay.cs

```diff
@@ -1,24 +1,21 @@
 ﻿using Newtonsoft.Json;
 
-namespace OsuSharp.Endpoints
+namespace OsuSharp.Entities
 {
-    public sealed class Replay : Endpoint
+    public sealed class Replay : EntityBase
     {
-        /// <summary>
-        ///     Replay's content
-        /// </summary>
-        [JsonProperty("content")]
-        public string Content { get; internal set; }
+        internal Replay() { }
 
         /// <summary>
-        ///     How is the content encoded
+        ///     The type of encoding used to encode <see cref="Content"/>
         /// </summary>
         [JsonProperty("encoding")]
         public string Encoding { get; internal set; }
```

Comment:
What kind of encoding is this? Could the `Encoding` class in stdlib be used here?

Reply:
It's usually base64 but knowing this api it could be anything so idk if I should use the enum there...